src/prefect/settings.py

```diff
@@ -503,16 +503,22 @@ def default_cloud_ui_url(settings, value):
     default=False,
 )
 """If `True`, places the API in test mode. This may modify
-behavior to faciliate testing. Defaults to `False`.
+behavior to facilitate testing. Defaults to `False`.
 """
 
+PREFECT_UNIT_TEST_MODE = Setting(
```

Comment:
I first tried using the existing `PREFECT_TEST_MODE` and `PREFECT_TEST_SETTING` (chained with `PREFECT_TEST_MODE`), but if I've understood correctly, `PREFECT_TEST_MODE` is set to True when you stand up a test server, and if that test server is setup behind a corporate network with deep packet inspection software which intercept the DNS (from original issue report), then that test server won't follow redirects.

Hope that makes sense – let me know if not :)

Reply:
Make sense to me, and it's nice to have a dedicated setting for unit tests.